io_scene_godot/converters/simple_nodes.py

```diff
@@ -106,6 +116,21 @@ def attribute_conversion(self):
         return self._light_attr_conv
 
 
+class CyclesLightNode(LightNode):
+    """Base class for godot light node based on a cycles engine light."""
+    _light_attr_conv = [
```

Comment:
I just checked the animation of cycles light, it is no trivial to handle, so `AttributeConvertInfo` has no help. Better to handle this two attributes as what we did to `shadow_enabled`, which is more readable.  Therefore, this new class may not be necessary.